api.py

```diff
@@ -27,59 +27,42 @@ def auth_client(self):
         response = requests.post(url, data=data, headers=headers)
         response_json = response.json()
 
-        return response_json['access_token']
+        self.token = response_json['access_token']
 
-    def post(self, token, url, data):
+    def authorize_header(self):
```

Comment:
NAB (Not a blocker): This is the kind of place where an _ would be appropriate - from the look of this function, it will only be used from inside the class, right?